website/content/v1.5/introduction/prodnotes.md

```diff
@@ -0,0 +1,310 @@
+manually configuring each node.
+If you do use this method, note that Talos requires a number of other
+kernel commandline parameters.
+See [required kernel parameters]({{< relref "../reference/kernel" >}}).
+
+Similarly, if creating [EC2 kubernetes clusters]({{< relref "../talos-guides/install/cloud-platforms/aws/" >}}), the configuration file can be passed in as `--user-data` to the `aws ec2 run-instances` command. See generally the [Installation Guide]({{< relref "../talos-guides/install" >}}) for the platform being deployed.
+
+### Separating out secrets
+
+When generating the configuration files for a Talos Linux cluster, it is recommended to start with generating a secrets bundle which should be saved in a secure location.
+This bundle can be used to generate machine or client configurations at any time:
+
+```sh
+talosctl gen secrets -o secrets.yaml
+```
+Now, we can generate the machine configuration for each node:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml <cluster-name> <cluster-endpoint>
+```
+
+Here, `cluster-name` is an arbitrary name for the cluster, used
+in your local client configuration as a label.
+It should be unique in the configuration on your local workstation.
+
+The `cluster-endpoint` is the Kubernetes Endpoint you
+selected from above.
+This is the Kubernetes API URL, and it should be a complete URL, with `https://`
+and port.
+(The default port is `6443`, but you may have configured your load balancer to forward a different port.)
+For example:
+
+```sh
+$ talosctl gen config --with-secrets secrets.yaml my-cluster https://192.168.64.15:6443
+generating PKI and tokens
+created controlplane.yaml
+created worker.yaml
+created talosconfig
+```
+
+### Customizing Machine Configuration
+
+The generated machine configuration provides sane defaults for most cases, but can be modified to fit specific needs.
+
+Some machine configuration options are available as flags for the `talosctl gen config` command,
+for example setting a specific Kubernetes version:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml --kubernetes-version 1.25.4 my-cluster https://192.168.64.15:6443
+```
+
+Other modifications are done with [machine configuration patches]({{< relref "../talos-guides/configuration/patching" >}}).
+Machine configuration patches can be applied with `talosctl gen config` command:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml --config-patch-control-plane @cni.patch my-cluster https://192.168.64.15:6443
+```
+
+> Note: `@cni.patch` means that the patch is read from a file named `cni.patch`.
+
+#### Machine Configs as Templates
+
+Individual machines may need different settings: for instance, each may have a
+different [static IP address]({{< relref "../advanced/advanced-networking/#static-addressing" >}}).
+
+When different files are needed for machines of the same type, there are two supported flows:
+
+1. Use the `talosctl gen config` command to generate a template, and then patch
```

Comment:
could be worth mentioning `talosctl apply-config --patch` also works

Reply:
we should probably try not to mention every option on other hand?